bin/travis.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+# Running Code Linter -- Requires @loopback/build so it's bootstrapped
+if [ $TASK = "code-lint" ]; then
+  echo "TASK => LINTING CODE"
+  lerna bootstrap --scope @loopback/build
+  npm run lint;
+
+# Commit Message Linter
+elif [ $TASK = "commit-lint" ]; then
+  echo "TASK => COMMIT MESSAGE LINTING"
+  commitlint-travis
+
+# Test LoopBack.io will build if changes made to @loopback/docs
+elif [ $TASK = "verify-docs" ]; then
+  echo "TASK => VERIFY DOCS"
+  if git diff --name-only --quiet $TRAVIS_BRANCH docs/; then
+    echo "No changes to @loopback/docs in this PR"
+    exit 0
+  else
+    echo "Testing @loopback/docs"
+    npm run verify:docs
+  fi
+
+# Mocha Tests
+else
+  echo "TASK => MOCHA TESTS"
+  lerna bootstrap
```

Comment:
Maybe `npm run postinstall` for consistency with other places where we defer the work to npm scripts?

IIRC, npm needs about ~50ms to execute the script, therefore your current solution is faster from performance view.

I don't have a strong opinion about this line, feel free to choose either way.